src/Commands/GenerateAliasesCommand.php

```diff
@@ -47,6 +46,7 @@ protected function createTsConfig(): void
     protected function writeAliases(): void
     {
         $tsconfig = json_decode(File::get($this->getTsConfigPath()), true);
+        $tsconfig['compilerOptions']['baseUrl'] = '.';
         $tsconfig['compilerOptions']['paths'] = collect(\config('vite.aliases'))
```

Comment:
In future, we should first check whether `baseUrl` is `null` or `'.'` before proceeding.
Also `paths` should be merged with existing paths.

Something like the following pseudo code:
```
if (baseUrl == null || baseUrl == '.' || baseUrl == './') {
    baseUrl = '.'
    paths = merge(paths, newPaths)
} else {
    print("Error: Can't automatically set paths when custom baseUrl is set.")
}
```

Reply:
> In future, we should first check whether baseUrl is null or '.' before proceeding.

What's the reason for this?

> Also paths should be merged with existing paths.

Good call. Thanks!

Reply:
When `baseUrl` is not `null`, `.` or `./`, then we can't assume that eg `{baseUrl}/resources` points to the location we intended. More likely in this case the user knows what they are doing.